tests/test_templater.py

```diff
@@ -157,8 +157,7 @@ def test_set_template_no_config_suffix_fails(): #pylint: disable=unused-variable
         args = [
             '-i', input_file,
             '-c', tmp_file.name,
-            '-d',
-            '-q',
+            '-d'
```

Comment:
Super tiny suggestion here -- a Python life hack, really. When managing lists over multiple lines, it's often helpful to include the comma after the last item for ease of supportability. If someone (or some tool) came in to rearrange the order or the list, or add another item to the list, there'd be no need to remember to add the extra comma here. Things move around in future iterations much more fluidly with that pesky extra comma.

Reply:
Good to know. I didn't realize we could have a comma at the end of a list.